mgr/config/v0.2.0-rhel7-root5.34.38-geant3-64b.config

```diff
@@ -0,0 +1,2 @@
+module use /cvmfs/star.sdcc.bnl.gov/star-spack/spack/share/spack/modules/linux-rhel7-x86_64/
```

Comment:
Looking at the naming convention...

`mgr/config/v0.2.0-rhel7-root5.34.38-geant3-64b.config`

I think this is fine... and I'll go ahead and approve.  But one thing to consider.  We are using the tag '64b' to indicate 64bits as we do with the setup script.  Then we are loading a module 'linux-rhel7-x86_64.   We could just as easliy adopt the convention

`mgr/config/v0.2.0-rhel7-x86_64-root5.34.38-geant3.config`

But even then... the specification of the operating system seems like it should be provided by the environment.

Reply:
I guess it is fine but seems a bit too verbose. Not sure why you want to specify the operating system 'rhel7'. I don't think it is going to change or we will support anything else. Also geant3 is a part of star-sw so, it is not really "optional". Even if geant4 is loaded at the same time there should be no conflict between the two...

By the way, the environment variable `USE_64BITS` is set when `star-env-...` is loaded

```
$ module show star-env-0.2.3-root-5.34.38
-------------------------------------------------------------------
/cvmfs/star.sdcc.bnl.gov/star-spack/spack/share/spack/modules/linux-rhel7-x86/star-env-0.2.3-root-5.34.38:

...
setenv           USE_64BITS 0
-------------------------------------------------------------------
```

```
$ module show star-env-0.2.3-root-5.34.38
-------------------------------------------------------------------
/cvmfs/star.sdcc.bnl.gov/star-spack/spack/share/spack/modules/linux-rhel7-x86_64/star-env-0.2.3-root-5.34.38:

...
setenv           USE_64BITS 1
-------------------------------------------------------------------
```